indexer/src/main/java/consumer/IndexConsumer.java

```diff
@@ -48,23 +50,28 @@ public IndexConsumer(RestHighLevelClient elasticSearchClient, KafkaConsumer<Inte
 
     private void run() {
         Logger logger = LogManager.getLogger(IndexConsumer.class);
+        AccountsServiceApi accountsServiceApi = new AccountsServiceApi();
+        ObjectMapper mapper = new ObjectMapper();
 
         // Subscribe to the topic.
         consumer.subscribe(Collections.singletonList(TOPIC));
```

Comment:
Not related to this PR, but let's do to topic also part of the configuration

Reply:
i'll do this in a separate branch